# This message carries all stuff necessary to plot
Header header
geometry_msgs/Point32[] rh_landmarks
geometry_msgs/Point32[] lh_landmarks
geometry_msgs/Point32[] pose_landmarks